sos/report/plugins/leapp.py

```diff
@@ -16,10 +16,12 @@ class Leapp(Plugin, RedHatPlugin):
     short_desc = 'Leapp upgrade handling tool'
 
     plugin_name = 'leapp'
-    packages = ('leapp', 'leapp-repository')
+    packages = ('leapp',)
+    files = ('/var/lib/leapp',)
 
     def setup(self):
         self.add_copy_spec([
+            '/etc/migration-results',
```

Comment:
does sosreport check whether the file exists? so it's not failing when the file is not present yet. just checking to be sure.

I wuold possibly extende it right away to include also:
```
/var/log/leapp/answerfile
/var/log/leapp/dnf-plugin-data.txt
```

Reply:
it does not fail when files are absent, no :)

Reply:
`dnf-plugin-data.txt` is already here, just scroll down a bit.
I  can add answerfile, sure